x-pack/test/functional/page_objects/lens_page.ts

```diff
@@ -596,6 +596,19 @@ export function LensPageProvider({ getService, getPageObjects }: FtrProviderCont
         await testSubjects.exists('lnsVisualOptionsButton');
       });
     },
+    async retrySetValue(
+      input: string,
+      value: string,
+      options = {
+        clearWithKeyboard: true,
+        typeCharByChar: true,
+      } as Record<string, boolean>
+    ) {
+      await retry.try(async () => {
+        await testSubjects.setValue(input, value, options);
+        expect(await (await testSubjects.find(input)).getAttribute('value')).to.eql(value);
```

Comment:
nit: it will be ideal to don't use `expect` in `page_object`.  You can probably use  `await testSubjects.existOrFail` like here: 

![image](https://user-images.githubusercontent.com/20072247/145874946-4b1d32fa-f6f3-4704-9dc3-1c389f0fef12.png)